scripts/docker-as-artifacts

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+set -e
+
+: ${ARTIFACTS_INSTALL:="/usr/bin/artifacts-install"}
+: ${ARTIFACTS_LOG_DIR:="/var/www/log"}
+
+# Artifacts variables
+export ARTIFACTS_DEST=${ARTIFACTS_DEST:="/usr/bin/artifacts"}
+export ARTIFACTS_KEY=${ARTIFACTS_KEY:=""}
+export ARTIFACTS_SECRET=${ARTIFACTS_SECRET:=""}
+export ARTIFACTS_BUCKET=${ARTIFACTS_BUCKET:="codeship-artifact"}
+export ARTIFACTS_REGION=${ARTIFACTS_REGION:="eu-west-1"}
+
+# Download Artifacts installer.
+curl -sL -o "${ARTIFACTS_INSTALL}" \
```

Comment:
can we pre-install this binary at build time (`Dockerfile`) ?

Reply:
Yes, we can install it at build time, could be better.

Reply:
How to do that ? Should I have a param `skip-upload` and use it during build time ? And where should I put the code to install stuff on Build time ?

Reply:
@WengerK here: https://github.com/antistatique/docker-php-dev/blob/dev/Dockerfile#L98

I'm not sure to exactly remember how to test the image without publishing it. @yannlugrin can you document this ?

Reply:
We have to move this code in `Dockerfile`:

```bash
# Download Artifacts installer.
curl -sL -o "${ARTIFACTS_INSTALL}" \
  "https://raw.githubusercontent.com/travis-ci/artifacts/master/install"

# Make it executable & install Artifacts.
chmod +x "${ARTIFACTS_INSTALL}"
"${ARTIFACTS_INSTALL}"
```

to something like that:
```Dockerfile
RUN set -ex; \
  \
  curl -fsSL  https://raw.githubusercontent.com/travis-ci/artifacts/master/install | bash
```

Could be added in the big install command.

Reply:
we also need `ARTIFACTS_DEST` into https://raw.githubusercontent.com/travis-ci/artifacts/master/install